src/components/common/form/CountrySelector.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Select from 'react-select';
+import PropTypes from 'prop-types';
+import countryList from 'react-select-country-list';
+
+const lists = countryList().getData();
+
+/**
+ * A select form component with list of countries.
+ *
+ * @component
+ * @example
+ * return (
+ *  <CountrySelector
+ *   selected = the selected item from the list
+ *   onChange = method invoked on selecting the item from the list.
+ *  />
+ * )
+ *
+ * @param  props
+ */
+const CountrySelector = props => {
```

Comment:
Instead of a generic `CountrySelector`, create a generic `Select` component that accepts data as a prop so that it can be used for multiple select components. 
Basically it should be an abstraction over `React Select`. 

Think of it this way, if we ever need to replace `React Select` with another library, only this particular file should have modifications while our entire app should work with the same `Select` component.

Reply:
Done